src/shoghicp/BigBrother/utils/InventoryUtils.php

```diff
@@ -1068,7 +1079,7 @@ public function onCraft(int $windowId) : void{
 		$resultRecipe = null;
 		foreach($this->shapedRecipes as $jsonResult => $jsonSlotData){
 			foreach($jsonSlotData as $jsonSlotMap => $recipe){
-				if($recipe->matchItems($inputSlotMap, $outputSlotMap)){
+				if($recipe->matchItems($inputSlotMap, $outputSlotMap)){//removed in 3.0.0-ALPHA12
 					$resultRecipe = $recipe;
 					break;
 				}
```

Comment:
matchItems, getResult functions removed in 3.0.0-ALPHA12
https://github.com/pmmp/PocketMine-MP/blob/547503e8f44d1475def28bda2c0ed068eb24e670/changelogs/3.0-alpha.md#crafting-1